v2/src/features/UploadList/UploadList.tsx

```diff
@@ -52,11 +52,44 @@ const generateErrorPayload = (error: AxiosError | Error) => {
   return errorPayload;
 };
 
+const LOCAL_STORAGE_KEY = 'sugar-share-files';
+
+const setOnLocalStorage = (key: string, value: any) => {
+  if (chrome.storage) {
```

Comment:
Making it future proof if I want to use this component on a WEB page instead of using chrome.storage it will use window.localStorage